pkg/thing/interactors/update_data.go

```diff
@@ -0,0 +1,69 @@
+package interactors
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+)
+
+var (
+	ErrNoSchema = fmt.Errorf("thing has no schema")
+	ErrDataInvalid = fmt.Errorf("data is incompatible with thing's schema")
+)
+
+// RequestData executes the use case operations to request data from the thing
```

Comment:
Opsie :grimacing:
This is actually `RequestData`'s comment :stuck_out_tongue:  

It should, instead, be something along these lines :point_down: 
```suggestion
// UpdateData executes the use case operations to update a thing's data
```

Reply:
The issue was fixed in between reviews, resolving this  :confetti_ball: